### ISSUE TITLE
Feature: Support for messages on damage rolls of features/spells with no attack rolls

### ISSUE THREAD

**Issue by Ikaguia** [kaelad02/adv-reminder#56]
Add support for a flag like `flags.adv-reminder.message.damage.other` for showing messages if there's no attack.
Currently you can only do it with `flags.adv-reminder.message.damage.all`, but it includes features/items with attacks.

**Comment by kaelad02** [comment]
When I added the messages for damage rolls, I just copied the critical role flags. I can easily get all the other action types and add message support for those.

**Comment by kaelad02** [comment]
I can get them from `CONFIG.DND5E.itemActionTypes`

**Comment by kaelad02** [comment]
Looks like they already work, lucked out there. For example, I created an effect with `flags.adv-reminder.message.damage.save` and it showed the message when I rolled damage on a spell with a saving throw (e.g. Sacred Flame).

There is an integration with DAE so these flags show up in the drop-down. I'll update that so they show up. I'll also double-check the documentation on the Wiki so it shows there too.
